Source/ablastr/fields/IntegratedGreenFunctionSolver.H

```diff
@@ -30,54 +31,92 @@ namespace ablastr::fields
      * @param[in] y y-coordinate of given location
      * @param[in] z z-coordinate of given location
      *
-     * @return the integrated Green function G
+     * @return the integrated Green function G in 3D
      */
     AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
     amrex::Real
-    IntegratedPotential (amrex::Real x, amrex::Real y, amrex::Real z)
+    IntegratedPotential3D (amrex::Real x, amrex::Real y, amrex::Real z)
     {
-        using namespace amrex::literals;
-
         amrex::Real const r = std::sqrt( x*x + y*y + z*z );
-        amrex::Real const G =
-            - 0.5_rt * z*z * std::atan( x*y/(z*r) )
-            - 0.5_rt * y*y * std::atan( x*z/(y*r) )
-            - 0.5_rt * x*x * std::atan( y*z/(x*r) )
-            + y*z*std::asinh( x/std::sqrt(y*y + z*z) )
-            + x*z*std::asinh( y/std::sqrt(x*x + z*z) )
-            + x*y*std::asinh( z/std::sqrt(x*x + y*y) );
+        amrex::Real const G = - 0.5_rt * z*z * std::atan( x*y/(z*r) )
+                              - 0.5_rt * y*y * std::atan( x*z/(y*r) )
+                              - 0.5_rt * x*x * std::atan( y*z/(x*r) )
+                              + y*z*std::asinh( x/std::sqrt(y*y + z*z) )
+                              + x*z*std::asinh( y/std::sqrt(x*x + z*z) )
+                              + x*y*std::asinh( z/std::sqrt(x*x + y*y) );
+        return G;
+    }
+
+
+    /** @brief Implements minus equation 58 in https://doi.org/10.1016/j.jcp.2004.01.008
```

Comment:
```suggestion
    /** @brief Implements equation 58 in https://doi.org/10.1016/j.jcp.2004.01.008
```
I think I understand what is meant here, but the term "minus equation" might be confusing. I think it is fine to simply say "equation 58".